tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -239,7 +239,7 @@ void test_wedge_map_generation_against_domain_helpers(
   const auto maps = wedge_coordinate_maps<Frame::Inertial>(
       inner_radius, outer_radius, inner_sphericity, outer_sphericity,
       use_equiangular_map, x_coord_of_shell_center, use_half_wedges);
-  for (size_t i = 0; i < maps.size(); i++) {
+  for (size_t i = 0; i < expected_coord_maps.size(); i++) {
```

Comment:
I assume this change indicates that you encountered a bug where these were not the same.  The function should presumably check that they match.